src/content/getusermedia/desktopcapture/extension/content-script.js

```diff
@@ -17,7 +17,7 @@ port.onMessage.addListener(function(msg) {
 });
 
 window.addEventListener('message', function(event) {
-	// We only accept messages from ourselves
```

Comment:
It's odd – also seems like Travis is complaining even though there's no complaint from JSCS locally. I'll look into it.